lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -137,6 +142,15 @@ negotiate_auto_portchannel:
       config_set_append: "%s negotiate auto"
       default_value: true
 
+port_channel:
+  kind: int
+  config_get: "show running interface all"
+  config_get_token: ['/^interface %s$/i', '/^channel-group (\d+)$/']
```

Comment:
You don't need to specify interface since the _template does that for you. 

e.g.
https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/interface.yaml#L10